tests/proxy/map_paging_predicate_test.py

```diff
@@ -0,0 +1,241 @@
+import os
+import unittest
+
+from tests.base import HazelcastTestCase
+from tests.util import configure_logging, get_abs_path, random_string
+from tests.custom_comparator import CustomComparator
+from hazelcast.serialization.predicate import PagingPredicate, is_greater_than_or_equal_to, is_less_than_or_equal_to, \
+    is_ilike
+from hazelcast import HazelcastClient
+from hazelcast.six import assertCountEqual
+from hazelcast.util import ITERATION_TYPE
+
+
+class MapPagingPredicateTest(HazelcastTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        configure_logging()
+
+    def setUp(self):
+        self.rc = self.create_rc()
+        self.cluster = self.create_cluster(self.rc, self._configure_cluster())
+        self.member1 = self.cluster.start_member()
+        self.member2 = self.cluster.start_member()
+        self.client = HazelcastClient()
+        self.map = self.client.get_map(random_string())
+
+    def tearDown(self):
+        self.map.destroy()
+        self.client.shutdown()
+        self.rc.exit()
+
+    """
+    Tests for proxy: comparator None
+    """
+    # Time out for assertTrueEventually set to 30. Could it be shorter?
```

Comment:
You can decrease the timeout to something like 5 seconds. Ideally, all tests in this file should be finished under a second. If not, something may be wrong